effectful/handlers/llm/providers.py

```diff
@@ -178,22 +226,23 @@ def push_current_text():
             if field_name is not None:
                 obj, _ = self.get_field(field_name, args, kwargs)
                 obj = self.convert_field(obj, conversion)
-
-                if isinstance(obj, Image.Image):
-                    assert not format_spec, (
-                        "image template parameters cannot have format specifiers"
-                    )
-                    push_current_text()
-                    prompt_parts.append(
-                        {
-                            "type": "image_url",
-                            "image_url": _pil_image_to_base64_data_uri(obj),
-                        }
+                # TODO(kg): should this use the serialize mechanism?
```

Comment:
Is this `TODO` a reference to your comment in Slack about formatting customization, or something else?

Reply:
yes, it's w.r.t the slack comment.

Reply:
removed the comment, keeping the discussion on github.